libs/components/src/lib/empty-state/empty-state.template.ts

```diff
@@ -0,0 +1,52 @@
+import {html, slotted, when} from '@microsoft/fast-element';
+import type { ViewTemplate } from '@microsoft/fast-element';
+import type {
+	ElementDefinitionContext,
+	FoundationElementDefinition,
+} from '@microsoft/fast-foundation';
+import { classNames } from '@microsoft/fast-web-utilities';
+import { Icon } from '../icon/icon';
+import type { EmptyState } from './empty-state';
+
+const getClasses = (x: EmptyState) => classNames('base',
+	['no-actions', x.slottedActionItems?.length === 0]
+);
+
+/**
+ * The template for the EmptyState component.
+ *
+ * @param context
+ * @public
+ */
+export const EmptyStateTemplate: (
+	context: ElementDefinitionContext,
+	definition: FoundationElementDefinition
+) => ViewTemplate<EmptyState> = (
+	context: ElementDefinitionContext
+) => {
+	const iconTag = context.tagFor(Icon);
+
+	return html<EmptyState>`
+		<div class="${getClasses}">
+			<div class="graphic">
+				<slot name="graphic">
+					<div class="icon-container">
+						<${iconTag} class="icon" name="${(x) => x.icon}" size="5"></${iconTag}>
+					</div>
+				</slot>
+			</div>
+			<div class="content">
+				${when(
+		(x) => x.headline,
+		html<EmptyState>`<h3 class="headline">${(x) => x.headline}</h3>`
```

Comment:
```suggestion
		html`<h3 class="headline">${(x) => x.headline}</h3>`
```
also - I know that in vivid-2 is `<h3>` but is it correct? Should it be a headline at all, and if so - do we need to allow the consumers to change it if its not the right headline?

Reply:
+1. maybe `<header>` as in some other components would just do?

Reply:
Sure, I've changed to a `<header>`
If I remove `<EmptyState>` though it doesn't typecheck  anymore so I've kept that